src/main/java/eu/enmeshed/model/requestItems/RequestItemGroup.java

```diff
@@ -1,17 +1,24 @@
 package eu.enmeshed.model.requestItems;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import java.util.List;
+import java.util.Map;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 import lombok.experimental.SuperBuilder;
 
-// @AllArgsConstructor
 @NoArgsConstructor
 @Getter
 @Setter
 @SuperBuilder
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "@type")
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
 public class RequestItemGroup extends RequestItem {
-
   private List<RequestItem> items;
+  private String title;
+  private String description;
+  private Map<String, String> metadata;
+  private Boolean requireManualDecision;
```

Comment:
The `requireManualDecision` property needs to be removed.

Reply:
removed